AdminWebsite/AdminWebsite.UnitTests/Services/HearingServiceTests.cs

```diff
@@ -95,6 +95,22 @@ public async Task should_send_multiday_confirmation_email_to_all_participants_ex
                     x => x.CreateNewNotificationAsync(It.Is<AddNotificationRequest>(r => r.ParticipantId != judge.Id)),
                     Times.Exactly(_hearing.Participants.Count(x => x.User_role_name.ToLower() != "judge")));
         }
+        
+        [Test]
+        public async Task should_not_create_notification_for_judge_email_when_groupid_is_null()
```

Comment:
Group Id will never be null. You can remove this test. If necessary, update the builders so it's not null and set to be the same value as the hearing ID unless it's representing a cloned hearing